data/iris.csv

```diff
@@ -0,0 +1,151 @@
+"sepal.length","sepal.width","petal.length","petal.width","variety"
```

Comment:
Please remove this file from your git repository.

This is a general available dataset.

Make use of a .gitignore file to fullfile this task.

Reply:
There are already multiple plots and a short section about EDA inside the Jupiter Notebook. Do you mean I should move it to the `docs` folder? Or should I write text in a `.tex`-file (#6 does not mention this)?

If I remove the dataset completely, others might not know that they have to import it themselves or where to get it in a format that is compatible with my code. If you don't want the .csv file inside the repository, I suggest replacing it with an import from a library.

```suggestion
from sklearn import datasets
iris = datasets.load_iris()
```

Reply:
Yes, please move the plots and the short section from the notebook to your docs.

Please do not use notebooks in future for carrying essential information. If you want, use them as a workplace.

I would prefer to place important future content to the docs folder and in your latex documentation.

Import py code should be placed in py - files. notebooks are not easy to handle in svc like git.

Reply:
This was quite confusing to me as the issue explicitly states to 'try out some plots and charts' in a Jupyter environment. I understood that the objective was to gain experience with Jupyter, but your subsequent instructions to transfer the notebook content to the docs left me uncertain. Additionally, the acceptance of the pull request https://github.com/philippabele/deep-groove-with-pytorch/pull/9, which appears to accomplish the same task as I did, added to my confusion.

Upon re-reading your comment, I'm not certain whether your suggestion to avoid using Jupyter with git applied to this particular pull request. If not, I would have expected this advice to be in an issue instead, as it initially appeared as though I was making an error while simply following your instructions